src/validation/userValidation.js

```diff
@@ -0,0 +1,19 @@
+import { check } from 'express-validator';
+
+const userValidation = [
+  check('firstName').not().isEmpty().withMessage('First name field cannot be empty.'),
```

Comment:
```suggestion
  check('firstName').not().isEmpty().trim().withMessage('First name field cannot be empty.'),
```

This would check for when users add empty spaces as well